color/src/main/java/com/afollestad/materialdialogs/color/DialogColorChooserExt.kt

```diff
@@ -35,35 +50,28 @@ fun MaterialDialog.colorChooser(
   subColors: Array<IntArray>? = null,
   @ColorInt initialSelection: Int? = null,
   waitForPositiveButton: Boolean = true,
+  allowCustomColor: Boolean = false,
+  supportCustomAlpha: Boolean = false,
+  @StringRes resTabGrid: Int? = null,
+  textTabGrid: String? = null,
+  @StringRes resTabCustom: Int? = null,
+  textTabCustom: String? = null,
```

Comment:
To make it easier to find these with code completion, it'd be better for them to be `tabCustomText`, `tabCustomTextRes`, `tabTextGrid`, `tabTextGridRes`, etc.

Reply:
done